src/components/Results.tsx

```diff
@@ -1,62 +1,90 @@
-import { Container, Row, Table } from "react-bootstrap";
-import dataSample from "../dataSample";
+import { useState, useEffect } from "react";
+import moment from "moment";
+import ClientSingleton from "../api/ClientSingleton";
 
-function renderData(
-	data: {
-		Index: string;
-		SegId: string;
-		NVoxels: string;
-		Volume_mm3: string;
-		StructName: string;
-		normMean: string;
-		normStdDev: string;
-		normMin: string;
-		normMax: string;
-		normRange: string;
-	},
-	index: number
-) {
-	return (
-		<tr key={index}>
-			<td>{data.Index}</td>
-			<td>{data.SegId}</td>
-			<td>{data.NVoxels}</td>
-			<td>{data.Volume_mm3}</td>
-			<td>{data.StructName}</td>
-			<td>{data.normMean}</td>
-			<td>{data.normStdDev}</td>
-			<td>{data.normMin}</td>
-			<td>{data.normMax}</td>
-			<td>{data.normRange}</td>
-		</tr>
-	);
-}
```

Comment:
This was from parsing a sample `aseg.stats` file, which we can reuse the parsing logic for the other .stats files later